### ISSUE TITLE
Wrong command syntax '--sort-by-required'

### ISSUE THREAD

**Issue by chakatz** [cytopia/docker-terraform-docs#55]
I'm a first-time user, but it seems to me that
`--sort-by-required` should be `--sort-by required`
It is wrong in several places in the readme.

https://github.com/cytopia/docker-terraform-docs/blob/master/README.md

I had to run this to figure it out the proper syntax
`docker run --rm -v $(pwd):/data cytopia/terraform-docs terraform-docs help`

**Comment by cytopia** [comment]
Good point. This is a relict from older versions when arguments where named differently. I will make it a PR to clean up the legacy commands.
